data/Gaussian/basicGaussian09/dvb_nmr.com

```diff
@@ -0,0 +1,28 @@
+%NProcShared=1
+%Mem=1GB
+#p NMR=(SpinSpin) B3LYP/sto-3g Symmetry=(Tight) Integral=(Ultrafine)
+
+dvb_nmr
+
+0, 1
+ C(Iso=13)       -1.4152533224   0.2302217854   0.0000000000
```

Comment:
Does Gaussian not automatically pick the most abundant isotope with nuclear spin > 0?

Reply:
Unfortunately it does not, without the specification it defaults to C12. Not entirely sure what the thinking there is but I'd guess it's historical. Gaussian only uses a single isotope definition for the whole calc (you cant request different isotopes just for NMR like you can with Orca) so I guess C12 makes more sense for other calc types that predate the NMR functions